### PR TITLE
chore(sdl): codemod it all

### PR DESCRIPTION
**Pull request by mattkrick**
This moves graphql/intranetSchema to graphql/private.

It refactors all the objects to SDL-driven development. 
It's big, but half of the changes are deletions. The other half are generated, no need to read through the resolve functions, they haven't changed a bit (except for the type errors that this caught 🎉 )

You can learn more here: https://www.loom.com/share/10ffe21d92014cb6b82f7d8aeb435516

TEST
- [ ] Goto /admin/graphql and query something on the private schema. it works

### CODE REVIEW

**Comment by mattkrick** [comment]
@Dschoordsch I experimented with a default mapper being void or never in order to force devs to explicitly write a `*Source` type definition. Here's my takeaways:

The current behavior has a default type == the GraphQL type. This works great for the following cases:
- The return type is a primitive. no mapper is necessary. 
- The Source type is a superset of the GraphQL type. no mapper is necessary.

Now the cool part is when the Source type is a subset of the GraphQL type, _typescript still throws an error_.

A good example is [user.ts](https://github.com/ParabolInc/parabol/blob/f18ec682dc27f3d6fce3173cda2d57e264154cb5/packages/server/graphql/private/queries/user.ts#L5):
- the user query returns the type `IUser`, which comes from postgres
- the user query is expected to return the typescript equivalent of `/graphql/types/User.ts`
- Therefore, we get an error because e.g. `IUser` does not include the field `timeline`.

This error forces the developer to add an entry to the `codegen#mappers` list. there's no implicit any in the logic chain :tada:

**Comment by mattkrick** [comment]
@JimmyLv thought this would be a fun PR for you to review! it's mainly a refactor to make our feature building a little more modern

File: packages/server/graphql/private/mutations/connectSocket.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import {InvoiceItemType, SubscriptionChannel} from 'parabol-client/types/constEnums'
+import adjustUserCount from '../../../billing/helpers/adjustUserCount'
```
Comment (blimmer) [inline packages/server/graphql/private/mutations/connectSocket.ts:2]: Should we consider using a typescript path to make these `../../../` imports simpler to read?

**Review by blimmer — APPROVED** [review]
I didn't review each individual resolver, but this is a great step to further leveraging the modern GraphQL ecosystem!

File: packages/server/lint-staged.config.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  // "*.{ts,tsx}": 'organize-imports-cli',
```
[the remark this answers is not in the document]
Comment (mattkrick) [reply packages/server/lint-staged.config.js:2]: ahhh i've gotta try that next time! this one was buggy & slow

**Comment by mattkrick** [comment]
@JimmyLv hows the review going?  any questions or comments?

**Comment by JimmyLv** [comment]
@mattkrick Hey Matt! This is really excellent! I watched the video earlier, to understand the intent behind this. 

This is also the first time, I learned how to write a jscodeshift script, understand the magic behind it!
As you mentioned in the loom video, will we move on to the public schema next? what and how different are these folders of public stuff?

**Comment by mattkrick** [comment]
yep, next step is the public schema!
the public schema is about 20x larger than this private schema, so that's why i wanted to make this private schema patterns perfect before we tackle the MASSIVE amount of work in the public schema

**Comment by JimmyLv** [comment]
Makes sense! I believe the `jscodeshift` for private schema could be adapted to public schema, it would be a lifesaver!

**Pull request merged**